qa/tasks/cephadm.py

```diff
@@ -447,6 +447,62 @@ def pull_image(ctx, config):
     finally:
         pass
 
+@contextlib.contextmanager
+def setup_ca_signed_keys(ctx, config):
+    # generate our ca key
+    cluster_name = config['cluster']
+    bootstrap_remote = ctx.ceph[cluster_name].bootstrap_remote
+    bootstrap_remote.run(args=[
+        'ssh-keygen', '-t', 'rsa', '-f', '/root/ca-key', '-N', '""'
```

Comment:
```
2023-07-09T18:44:33.474 DEBUG:teuthology.orchestra.run.smithi006:> ssh-keygen -t rsa -f /root/ca-key -N '""'
2023-07-09T18:44:33.664 INFO:teuthology.orchestra.run.smithi006.stderr:Saving key "/root/ca-key" failed: Permission denied
2023-07-09T18:44:33.665 DEBUG:teuthology.orchestra.run:got remote process result: 1
```
https://pulpito.ceph.com/adking-2023-07-09_18:23:06-orch:cephadm-wip-adk4-testing-2023-07-08-2258-distro-default-smithi/7331648/

Normal deployment tests are passing again, just need to get this ca signed key deployment to actually work now

Reply:
probably just need `sudo` here